### PR TITLE
OCSADV-274: Created spec file to build rpm package.

### PR DESCRIPTION
**Pull request by jluhrs**
The spec file will create an rpm that install the dhs-http files as a service for the axis2 HTTP server.

### CODE REVIEW

File: DhsTranslator/DhsTranslator.spec [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+%define release __auto__
+%define repository gemini
+
+%define debug_package %{nil}
+
+Summary: %{name} Package
+Name: %{name}
+Version: %{version}
+Release: %{release}.%{dist}.%{repository}
+License: GPL
```
Comment (cquiroz) [inline DhsTranslator/DhsTranslator.spec]: Is this true?

File: DhsTranslator/DhsTranslator.spec [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Source:%{name}-%{version}.tar.gz
+Group: Gemini
+Source0: %{name}-%{version}.tar.gz
+BuildRoot: /var/tmp/%{name}-%{version}-root
+BuildArch: %{arch}
+Prefix: %{_prefix}
+## You may specify dependencies here
+BuildRequires:axis2c-unofficial-devel%{?_isa} dhsClient-devel%{?_isa} drama-devel%{?_isa}
+Requires: axis2c-unofficial%{?_isa} drama%{?_isa} dhsClient%{?_isa}
```
Comment (cquiroz) [inline DhsTranslator/DhsTranslator.spec:23]: I may not remember this too well, will this require the `axis2c-unofficial` rpm from our yum repo?
Is it there?

Comment (jluhrs) [inline DhsTranslator/DhsTranslator.spec:23]: Yes, and yes.

File: DhsTranslator/DhsTranslator.spec (hunk 1 above)
Comment (jluhrs) [inline DhsTranslator/DhsTranslator.spec]: The dangers of copy-and-paste.

**Comment by cquiroz** [comment]
Seems good, though I wonder why there are other code changes besides the spec file
I don't know about the licensing, though being an internal package it may not be relevant.
Perhaps @arturog8m can comment on the license

**Comment by arturog8m** [comment]
Right, we can't state this is GPL as it is not. Please change it to BSD-3 which is more flexible and what we are using for everything else.

**Pull request merged**